fe/dist/bundle.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
- what this file for?

Reply:
it was installed when I am solving a problem with webpack.config.js , I can remove it po